src/Controls/src/Core/HandlerImpl/Button/Button.cs

```diff
@@ -16,8 +16,15 @@ public partial class Button
 		public static IPropertyMapper<IButton, ButtonHandler> ControlsButtonMapper = new PropertyMapper<Button, ButtonHandler>(ButtonHandler.Mapper)
 		{
 			[nameof(ContentLayout)] = MapContentLayout,
+#if ANDROID
+			[nameof(ImageSource)] = MapImageSource,
+#endif
 #if IOS
 			[nameof(Padding)] = MapPadding,
+			[nameof(ImageSource)] = MapImageSource,
+#endif
+#if MACCATALYST
+			[nameof(ImageSource)] = MapImageSource,
```

Comment:
I will implement Tizen as a follow-up PR. Please add `MapImageSource()` to `Button.Tizen.cs` as a `[MissinMapper]`

```cs
        [MissingMapper]
        public static void MapImageSource(IButtonHandler handler, Button button) { }
```

This will eliminate unnecessary branches and keep the code more concise as below.

```cs
		public static IPropertyMapper<IButton, ButtonHandler> ControlsButtonMapper = new PropertyMapper<Button, ButtonHandler>(ButtonHandler.Mapper)
		{
			[nameof(ContentLayout)] = MapContentLayout,
			[nameof(ImageSource)] = MapImageSource,
#if IOS
			[nameof(Padding)] = MapPadding,
#endif
#if WINDOWS
			[nameof(IText.Text)] = MapText,
#endif
			[nameof(TextTransform)] = MapText,
			[nameof(Text)] = MapText,
			[nameof(Button.LineBreakMode)] = MapLineBreakMode,
		};
```